Gemfile.lock

```diff
@@ -7,12 +7,13 @@ GIT
 
 GIT
   remote: https://github.com/moneyadviceservice/dough.git
-  revision: cf4e4cbd5ee86ef49b2db16fc44dde1ed9f2c0f4
+  revision: f696d53ddac5f73d64a5d32ecbe03fd9d0b9298e
   specs:
-    dough-ruby (5.7.0)
+    dough-ruby (5.24.1)
```

Comment:
had to update dough-ruby again as it couldn't find this version.
latest is `5.27`